src/garage/np/algos/meta_algorithm.py

```diff
@@ -0,0 +1,35 @@
+"""Interface of Meta-RL ALgorithms."""
+import abc
+
+from garage.np.algos import RLAlgorithm
+
+
+class MetaRLAlgorithm(RLAlgorithm):
+    """Base class for Meta-RL Algorithms."""
+
+    @abc.abstractmethod
+    def new_exploration_policy(self):
```

Comment:
i'm not sure about this name because it implies (to me) that a `__new__` is being called somewhere. can it be called `get_exploration_policy` instead?

if getting the exploration policy never implies non-trivial work, you could make this a property (`exploration_policy`). however i'm not sure that's necessarily the case for all algorithms.